<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="menuLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 51, 102, 150); 
color: lightblue; 
padding: 5px; 
border: 1px solid #003366; </string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>630</width>
      <height>630</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(50, 0, 50, 150); 
color: lime; 
font: bold 12pt &quot;Source Code Pro&quot;; 
padding: 5px; 
border: 1px solid #330033; </string>
    </property>
    <property name="text">
     <string>hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="menuEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>320</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(60, 60, 120, 150);
color: #CCCCCC; 
font: bold 12pt &quot;Consolas&quot;; 
border: 2px solid #003366;
padding: 6px; </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
